plugin/pkg/scheduler/generic_scheduler.go

```diff
@@ -165,6 +165,7 @@ func prioritizeNodes(pod *api.Pod, podLister algorithm.PodLister, priorityConfig
 		}
 	}
 	for host, score := range combinedScores {
+		glog.V(10).Infof("Host %s Score %d", host, score)
```

Comment:
It's OK, I just remember working in Kubelet or some controller, where we put logs of pretty much every step in V(4).